### DISCUSSION TITLE
Renaming the main branch

### DISCUSSION THREAD

**Discussion by Lamparter** [IviriusCommunity/Rebound#16] [Polls]
Currently, the main branch is `master`. It's no longer the default branch name and it's common practice to use `main` instead. 

_Copied from Esmerelda repo_

| Branch name | Origin | Description | Reason for implementation
|--------|--------|--------|--------|
| `master` | @torvalds | The default branch used by Git itself | _N/A_: Speculative |
| `main` | @octocat | The default branch used on GitHub | _N/A_: Speculative |
| `insider` | @josephbeattie | This is an excellent branch name idea | Fits in well with existing semantic versioning conventions (River2 Convention) |
| `dev` | @mousemaid | This is the previous branch name idea proposed for the Esmerelda-Community organisation. | Used originally across existing Riverside Valley projects and follows legacy River1 Convention |
| `bloom` | @josephbeattie | This is a pretty idea for a naming convention and I really like the idea that the main branch would be called _'bloom'_ as a standard. | Business logic |

> _Currently looks like the best branch name is either `bloom` or `insider`_ 🤔

_cc. @errortek @IviriusMain_

**Comment by IviriusMain** [comment]
Make it Alpha since that's the current version we're all working on

**Reply by Lamparter** [reply]
No. (DM)

**Comment by errortek** [comment]
I feel like the branch for the **stable** release should be called "main", and for development, "dev", "alpha", "beta" depending on development stage

**Reply by Lamparter** [reply]
Same reason as I said no to @IviriusMain, let me explain in the discord server

**Comment by Lamparter** [comment]
`main` it is!
